app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :likes
+  has_many :comments
+
+  def update_post_counter
+    user.update(posts_counter: Post.where(user_id: user).count)
+  end
+
+  def last_comments
+    comments.last(5)
+  end
```

Comment:
- [OPTIONAL] The idea of most recent is that we should get the result starting from the very last. Considering this, this code can be refactored as:

```ruby
  def last_comments
    comments.order('created_at Desc').limit(5)
  end
```

OR

```ruby
  def last_comments
    comments.order('created_at Desc').first(5)
  end
```

Reply:
Well solved